pipelines/nextflow/modules/repeat_modeler/check_and_download_rmlibrary.nf

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env nextflow
+/*
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+process CHECK_AND_DOWNLOAD_RMLIBRARY {
+    tag "$gca:fetch_file_repeatmodeler"
```

Comment:
Just a general comment but since we see tags next to process names I dont think we need to repeat the purpose of the task in the tag. I have frustrated myself by doing this recently and still haven't fixed it. Perhaps just `$gca` would be sufficiently informative without crowding the UI

But perhaps you like longer tags for reasons I dont know so up to you.